### DISCUSSION TITLE
Excluding files with links?

### DISCUSSION THREAD

**Discussion by JesusFromTheBlock** [Vinzent03/find-unlinked-files#64] [Q&A] [answered]
Hi, I'm trying to find truly orphaned files - completely unconnected files with neither backlinks or links out. The orphaned feature finds files without backlinks. But I'd like to exclude files from these results that have links out (i.e. files that are connected, just not via a backlink). Is that possible some how?  Otherwise its a really helpful plugin so thanks a lot.

**Comment by Vinzent03** [comment, marked as answer]
You can use the "Exclude links" setting under the "Settings for finding orphaned files" by setting the value to `*` as documented in description of that setting.

**Comment by JesusFromTheBlock** [comment]
Thanks, I'd searched the documentation and setting but didn't see that
little asterix. Thanks and sorry for asking something answered.

**Reply by Vinzent03** [reply]
No problem, great it works for you now!
